tests/pkg/notifications/operations/execute-status-report.go

```diff
@@ -112,6 +113,11 @@ func (o *ExecuteStatusReportOperation) Execute(t *testing.T, ctx context.Context
 	response, err := o.client.Do(request)
 	require.NoError(t, err)
 	require.Equal(t, o.statusCode, response.StatusCode)
+	if o.statusCode > 299 {
```

Comment:
Could we add a comment here, for example: `// Log the unsuccessful HTTP responses` or something similar?